ui/raidboss/data/03-hw/deep_dungeon/the_palace_of_the_dead_general.ts

```diff
@@ -0,0 +1,184 @@
+      // duplicate item message: https://xivapi.com/LogMessage/7222?pretty=true
+      // en: You return the pomander of ${pomander} to the coffer. You cannot carry any more of that item.
+      type: 'SystemLogMessage',
+      netRegex: { id: '1C36' },
+      infoText: (_data, matches, output) =>
+        output.duplicate!({ pomander: pomanders[parseInt(matches.param1, 16) - 1] }),
+      outputStrings: {
+        duplicate: {
+          en: '${pomander} duplicate',
+        },
+      },
+    },
+    // ---------------- Floor Notifications ----------------
+    {
+      id: 'PotD General Cairn of Return',
+      // resurrects dead players
+      // Cairn of Return activation message: https://xivapi.com/LogMessage/7242?pretty=true
+      // en: The Cairn of Return begins to glow!
+      type: 'SystemLogMessage',
+      netRegex: { id: '1C4A', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Cairn of Return activated',
+        },
+      },
+    },
+    {
+      id: 'PotD General Cairn of Passage',
+      // portal to transfer between floors
+      // Cairn of Passage activation message: https://xivapi.com/LogMessage/7245?pretty=true
+      // en: The Cairn of Passage is activated!
+      type: 'SystemLogMessage',
+      netRegex: { id: '1C4D', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Cairn of Passage activated',
+        },
+      },
+    },
+    {
+      id: 'PotD General Accursed Hoard',
+      // Accursed Hoard on this floor message: https://xivapi.com/LogMessage/7272?pretty=true
+      // it seems that this SystemLogMessage isn't sent if it occurs immediately after transference between floors
+      // trigger off of the GameLog message instead
+      // en: You sense the Accursed Hoard calling you...
+      type: 'GameLog',
+      netRegex: { line: 'You sense the Accursed Hoard calling you\.\.\..*?', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Accursed Hoard on this floor',
+        },
+      },
+    },
+    {
+      id: 'PotD General Floor Enchantment',
+      // 440 = Blind (accuracy reduced)
+      // 441 = HP Penalty (maximum HP reduced)
+      // 442 = Damage Down (damage dealt reduced)
+      // 443 = Haste (weaponskill recast, spell recast, and auto-attack delay reduced)
+      // 444 = Amnesia (can't use abilities)
+      // 445 = HP & MP Boost (maximum HP & MP increased)
+      // 446 = Item Penalty (can't use items or pomanders)
+      // 447 = Sprint Penalty (can't use Sprint)
+      // 448 = Knockback Penalty (knockback and draw-in effects for enemies and players disabled)
+      // 449 = Auto-heal Penalty (out-of-combat passive HP regeneration disabled)
+      // TODO: trigger might overlap when multiple enchantments exist on the same floor
```

Comment:
For this, you might want to do a collect --> call --> cleanup construction. You can see a clean example in Sephirot EX/Unreal, starting at line 345. https://github.com/quisquous/cactbot/blob/0e6d23b401f82d48412f3b1f7232de4a7156b55d/ui/raidboss/data/06-ew/trial/sephirot-un.ts#L345-L390

Reply:
Thanks for the idea, but I ended up removing the enchantment triggers per quisquous's request.  I'll keep this pattern in mind for the future.